### PR TITLE
home screen test added

### PR DESCRIPTION
**Pull request by MattWay224**
home screen test added

### CODE REVIEW

**Comment by MattWay224** [comment]
Still there is need in fix, cause test cannot find bottomnavigationbar

File: test/home_screen_widget_test.dart [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+  testWidgets('renders HomeScreen and displays AppBar title', (WidgetTester tester) async {
+    await tester.pumpWidget(
+      const ProviderScope(
+        child: MaterialApp(
+          home: HomeScreen(),
+        ),
+      ),
+    );
+
+    expect(find.text('Run Tracker'), findsOneWidget);
```
Comment (dinaraparanid) [inline test/home_screen_widget_test.dart:20]: Ломается из-за локализации

**Review by dinaraparanid — CHANGES_REQUESTED** [review]

File: test/home_screen_widget_test.dart (hunk 1 above)
Comment (MattWay224) [reply test/home_screen_widget_test.dart:20]: пофикшу

**Comment by MattWay224** [comment]
Still there is need in fix

**Review by dinaraparanid — APPROVED** [review]

**Pull request merged**